src/config.js

```diff
@@ -0,0 +1,6 @@
+const LISTBANNERBGSRC =
+  "https://images.unsplash.com/photo-1536924940846-227afb31e2a5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1047&q=80";
+
+const API = "http://localhost:3000";
```

Comment:
따로 아래에서 합쳐서 export 하지 않고 아래와 같이 사용해주실 수 있습니다. 

```suggestion
export const API = "http://localhost:3000";
```